redmine_zulip/lib/zulip_hooks.rb

```diff
@@ -104,10 +117,10 @@ def send_zulip_message(content, project)
 
         Rails.logger.info "Forwarding to Zulip: #{data['content']}"
 
-        http = Net::HTTP.new("api.zulip.com", 443)
+        http = Net::HTTP.new(zulip_server(), zulip_port())
         http.use_ssl = true
 
-        req = Net::HTTP::Post.new("/v1/messages")
+        req = Net::HTTP::Post.new("/api/v1/messages")
```

Comment:
So the way API URLs work is that they are e.g.:
* api.zulip.com/v1/messages on zulip.com
* zulip.example.com/api/v1/messages for random Zulip servers
So to avoid breaking existing usage against zulip.com, I think we might want to make a function `zulip_api_basename()` that returns either `/v1/` or `/api/v1/` depending on whether `zulip_server` was `api.zulip.com`